migration-engine/connectors/sql-migration-connector/src/pair.rs

```diff
@@ -25,6 +25,17 @@ impl<T> Pair<T> {
         (&self.previous, &self.next)
     }
 
+    pub(crate) fn interleave<F, I, O>(&self, f: F) -> impl Iterator<Item = Pair<O>>
```

Comment:
Abstract methods like this would gain a lot from some documentation...

Reply:
can do!

Reply:
pushed a commit with a docstring —_feedback on the docstring is very welcome